### DISCUSSION TITLE
"Your feed" section instead of current feeds

### DISCUSSION THREAD

**Discussion by akshitagupta15june** [community/community#37757] [Other Feature Feedback, Questions, & Ideas]
"Your feed" section will be wonderful as we will be able to see the updates in our own repositories also the stars, followers as well as a forks.

**Comment by jfullstackdev** [comment]
yup, I think this is requested many times, 

anyway, as of now, you can create an Org with just you as the member, your dashboard is all about the repos in that Org, every detail of it

**Comment by akshitagupta15june** [comment]
yes, i can see
I have a org but only i can see is the followers details not any updation

**Comment by jfullstackdev** [comment]
this is what it looks like 

![image](https://user-images.githubusercontent.com/47092464/198875741-4786f1ee-ce2d-480c-8678-1501ff3b17d4.png)

if you don't have activities for so long, that will be empty

**Comment by akshitagupta15june** [comment]
fine! thanks
